src/contracts/Output/Generator.php

```diff
@@ -197,12 +197,10 @@ public function valueElement(string $name, $value): void
     }
 
     /**
-     * Start value element.
-     *
-     * @param string $name
-     * @param string $value
+     * @phpstan-param scalar $value
```

Comment:
Afaik `null` is a valid element value, correct?
```suggestion
     * @phpstan-param scalar|null $value
```

Reply:
> Afaik `null` is a valid element value, correct?

So... you're saying null ain't scalar? Copy.

Reply:
If you think about it, it makes sense for PHPStan to differentiate between `scalar` and `null` values. Oftentimes you'd expect a function to receive or return a scalar value, but `null` remaining unnacceptable, as a "lack of any value".